tests/framework/utils/k8s_helper.go

```diff
@@ -64,6 +62,16 @@ const (
 	hostnameTestPrefix = "test-prefix-this-is-a-very-long-hostname-"
 )
 
+// GetCmd returns kubectl or oc if env var rook_test_openshift is
+// set to true
+func GetCmd() string {
+	CliCmd := "kubectl"
```

Comment:
Local variable should not start with capital C

Reply:
Good Catch. Done!